### PR TITLE
Implementing requested fixes

### PR DESCRIPTION
**Pull request by moxvallix**
> Appearance editor should start with the equipped items/colors already selected in ui

### CODE REVIEW

File: src/ui_elements/appearance/color_picker.gd [hunk 1]
```diff
@@ -11,19 +11,36 @@ var colorMapImage: StreamTexture
...
+# --Public Functions--
+
+# Takes a position, scales it and displays the preview box
+func showPreview(pos: Vector2) -> void:
+	call_deferred("_showPreview", pos) # Call deferred runs it when it is ready
+
+# Returns the image scale of the color picker
+func getImageScale() -> Vector2:
+	var output = call_deferred("_getImageScale") # Call deferred runs it when it is ready
+	return(output)
```
Comment (nicemicro) [inline src/ui_elements/appearance/color_picker.gd]: does the call_deferred work actually with the return value? I wonder how does this work.
Also, please make sure this runs if the window is resized.

File: src/autoloads/appearance.gd [hunk 2]
```diff
@@ -1,10 +1,11 @@
 extends Node
 
-signal applyConfig # Apply config for menu character
+signal configUpdated # Apply config for menu character
```
Comment (nicemicro) [inline src/autoloads/appearance.gd]: "config" is not the good word here. I definitely wouldn't associate from the word "config" to the appearance.
also, signals are function calls, so verb + noun makes sense for me, but that could be debated.

changeOwnOutfit or changeOwnAppearance would be better choices IMHO even if longer words.

**Review by nicemicro — COMMENTED** [review]
see in-line comments.

File: src/autoloads/appearance.gd (hunk 2 above)
Comment (moxvallix) [reply src/autoloads/appearance.gd]: ChangeOwnOutfit wont work as outfit is separate to colours, i jad config meaning outfit + colours, but i guess we can change to updateAppearance.

File: src/ui_elements/appearance/color_picker.gd (hunk 1 above)
Comment (moxvallix) [reply src/ui_elements/appearance/color_picker.gd]: the scale is determined from _draw(), so it should always work no matter the window shape or size.

Comment (moxvallix) [reply src/ui_elements/appearance/color_picker.gd]: I deleted the getImageScale anyways because it wasn't really useful and didn't work.

**Review by nicemicro — APPROVED** [review]
It fixed what it had to fix.

**Pull request merged**